src/components/Board/Action.vue

```diff
@@ -82,6 +86,13 @@ export default {
     this.$refs.qkbMessageInput.focus()
   },
 
+  // focus text field on new messages (if appropriate)
+  watch: {
+    msgCount (newVal) {
+      this.$refs.qkbMessageInput.focus()
```

Comment:
(Sorry, don't have my laptop back so can't test directly)

Reply:
It probably does make sense to test... although I implemented that line and when I was testing, it failed pretty gracefully and didn't do anything when the text field was disabled. I thought it might make more sense to just have a blanket focus event rather than adding a bunch of conditionals, unless you think there might be some other negative effects

Reply:
Fair enough. Let's run with it and fix later if it breaks anything.